README.md

```diff
@@ -1,15 +1,51 @@
+<div id="top"></div>
+
 # HASH
 
 This repository contains HASH's open-source and public code, documentation, and other key resources, including:
 
-- `packages/hash`: The codebase for HASH ([Info](https://hash.ai/platform/index)) - coming soon, available under an Apache 2.0 compatible license
+- `packages/hash`: The codebase for HASH - an open-source, data-centric workspace based on the [Block Protocol](https://github.com/blockprotocol/blockprotocol), coming soon
 - `packages/engine`: The codebase for [hEngine](packages/engine) ([Info](https://hash.ai/platform/engine)) - available under the Server Side Public License
 - `packages/engine/stdlib`: The [standard library](packages/engine/stdlib) of helper functions available in all HASH simulations
 - `resources/docs`: A user guide to the whole HASH platform ([hash.ai/docs](https://hash.ai/docs))
 - `resources/glossary`: A glossary of terms explaining common concepts relevant to the use of HASH ([hash.ai/glossary](https://hash.ai/glossary))
 
-Please see:
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+
+<!-- CONTRIBUTING -->
```

Comment:
[optional]: I guess we can remove comments that replicate headers. When using an editor with syntax highlighting, headers look distinguishable enough. Comments don’t affect the result in any way AFAIU.